### ISSUE TITLE
The "Angle value" changes with the cursor location; but it should not. 

### ISSUE THREAD

**Issue by sichendart** [tomography/xfluo#48]

**Comment by FabricioSMarin** [comment]
Is this the ‘value’ label on the upper right-hand corner? I may have inadvertently labeled it as ‘Angle value’ at some point. That label should correspond to the pixel value so I’ll rename it to ‘pixel value’ to avoid confusion.

**Comment by sichendart** [comment]
Yes, it is the value on the upper right-hand corner. Yes, 'pixel value' makes sense.

**Comment by sichendart** [comment]
It seems that when the cursor is off the image. The pixel value is gone, but not the x or y pixel. I suggest to change the x/y pixel values to have the save behavior as the pixel value. Also, the angle at the bottom does not seem to update. 
<img width="591" alt="Screen Shot 2020-02-09 at 1 05 10 PM" src="https://user-images.githubusercontent.com/9825296/74108175-27244c80-4b3d-11ea-8d7a-ffb4fbc9a718.png">

**Comment by FabricioSMarin** [comment]
I'll change the x_pos and y_pos to have the same behavior, but the unchanging angle value is strange. This does not seem to be an issue on linux so I'll try to replicate the issue on my mac virtual machine. If the issue cannot be replicated, we'll need to meet and troubleshoot this together.
